### DISCUSSION TITLE
Not able to write data into mysql db using tabulator

### DISCUSSION THREAD

**Discussion by Swarnitha-eluru** [frictionlessdata/frictionlessdata.io#597] [Q&A] [answered]
I tried the code to read data from the source and write to mysql database.
![image](https://user-images.githubusercontent.com/46745717/87136649-69eb7b80-c2b9-11ea-8cc4-5771661b94a6.png)
I got the result like this
![image](https://user-images.githubusercontent.com/46745717/87136720-84255980-c2b9-11ea-9fb0-e313023612bf.png)
please help to resolve this

**Comment by roll** [comment, marked as answer]
Hi @Swarnitha-eluru,

I'm not around this week so I can't test it but in general, we're working on a major improvement of this functionality (i.e. SQL read/write) that will be hopefully available soon.

At the moment, I would recommend using a native SQL writer.

**Comment by Swarnitha-eluru** [comment]
okay thanks for the update
